src/util/fs.ts

```diff
@@ -47,3 +47,39 @@ export function glob(pathName: string): Promise<string[]> {
     });
   });
 }
+
+/**
+ * Write contents to file.
+ *
+ * @param {string} filepath
+ * @param {any} data
+ * @returns {Promise<string>}
+ */
+export function write(filepath: string, data: any): Promise<void> {
+  return new Promise((resolve, reject) => {
+    fs.writeFile(filepath, data, err => {
+      if (err) {
+        return reject(err);
+      }
+
+      return resolve();
+    });
+  });
+}
+
+/**
+ * Remove the file.
+ *
+ * @param { string } filepath
+ */
+export function remove(filepath: string): Promise<string> {
+  return new Promise((resolve, reject) => {
+    fs.unlink(filepath, err => {
+      if (err) {
+        return reject(err);
+      }
+
+      return resolve(`${filepath} was removed`);
```

Comment:
No need to return a string message on success. Just a `boolean` value `true` or even `void` would do.
```suggestion
      return resolve();
```